### ISSUE TITLE
[Enhancement] Organize fields on on-groups blocks

### ISSUE THREAD

**Issue by kisonay** [angell-co/Spoon#12]
Possible enhancement. I have a matrix field that is rather small (3 blocks) and would like to use Spoon to organize the fields on each block.

I can't organize the fields unless the blocks are grouped together, which seems a little odd with 1 block in each group.

It would be nice to allow field organization for non-grouped blocks.

**Comment by joshangell** [comment]
I agree, its an unfortunate side effect of the field layouts depending on the block type groupings in the database.

**Comment by jsunsawyer** [comment]
Yep. This was confusing as I thought I was missing an option. Installed Spoon simply for this feature.
